go.mod

```diff
@@ -32,15 +31,13 @@ require (
 	github.com/onsi/gomega v1.9.0 // indirect
 	github.com/privacybydesign/gabi v0.0.0-20210520084001-626ad7db13a6
 	github.com/shopspring/decimal v1.2.0 // indirect
-	github.com/pelletier/go-toml v1.2.0 // indirect
-	github.com/sietseringers/cobra v1.0.1-0.20200909200314-c50c3838234b
 	github.com/sietseringers/go-sse v0.0.0-20200801161811-e2cf2c63ca50
```

Comment:
Any chance we could insource this, too? The original code (https://github.com/andrewstuart/go-sse) hasn't been updated for quite a while.

Have we ever contacted Andrew Stuart?

Reply:
I have not.

I don't think it makes much sense to try to insource this. Of the original code of that repo, there is little left in my fork; I have even added a design goal that he apparently did not have (namely, adher to the SSE spec). The code is still fairly simple and small, however, so instead of rebasing onto changes from upstream (which don't appear to be coming), I was just planning to maintain my fork myself (if and when necessary).